src/garage/torch/algos/maml.py

```diff
@@ -305,7 +295,7 @@ def _compute_policy_entropy(self, task_samples):
         """
         obs = torch.stack([samples.observations for samples in task_samples])
         # pylint: disable=protected-access
-        entropies = self._inner_algo._compute_policy_entropy(obs)
+        entropies = self._inner_algo.compute_policy_entropy(obs)
```

Comment:
it seems the pylint-disable above should no longer be necessary, but why was `compute_policy_entropy` changed to public?

Reply:
I can remove the pylint-disable
And I change the function to public because the other (MAML class) uses the function.
(Even MAML is not a child class.)